### PR TITLE
feat: add orange-heart-monokai theme

### PR DESCRIPTION
**Pull request by joriewong**

### CODE REVIEW

**Comment by abnerlee** [comment]
Seems it already exists https://theme.typora.io/theme/OrangeHeart/

**Comment by joriewong** [comment]
> Seems it already exists https://theme.typora.io/theme/OrangeHeart/

code style is monokai

**Comment by abnerlee** [comment]
I see, maybe you can change it from "theme" to "fork" following https://github.com/typora/typora-theme-gallery/pull/165

**Comment by joriewong** [comment]
> I see, maybe you can change it from "theme" to "fork" following #165

done.

File: _posts/fork/2021-03-22-OrangeheartMonokai.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+layout: theme
+title: OrangeheartMonokai
+category: theme
```
Comment (abnerlee) [inline _posts/fork/2021-03-22-OrangeheartMonokai.md]: We need to change this from theme to fork as well.

And maybe put a link in https://theme.typora.io/theme/OrangeHeart/

File: _posts/theme/2020-06-14-OrangeHeart.md [hunk 2]
```diff
@@ -11,10 +11,18 @@ typora-root-url: ../../
...
 
 ![english](/media/theme/orangeheart/english.png)
 
-![chinese](/media/theme/orangeheart/chinese.png)
\ No newline at end of file
+![chinese](/media/theme/orangeheart/chinese.png)
+
+## Orange Heart Monokai
+
+Views details on this theme on the [homepage](https://github.com/joriewong/typora-theme-orange-heart-monokai).
```
Comment (joriewong) [inline _posts/theme/2020-06-14-OrangeHeart.md:24]: already put a link

**Pull request merged**